.storybook/preview.tsx

```diff
@@ -18,10 +18,12 @@ OriginalNextImage.propTypes = {
 OriginalNextImage.defaultProps = {
   unoptimized: true,
 };
-Object.defineProperty(NextImage, "default", {
-  configurable: true,
-  value: (props) => <OriginalNextImage {...props} unoptimized />,
-});
```

Comment:
I refactored this similarly: https://github.com/oaknational/Oak-Web-Application/pull/2171/files#diff-e201de4e2ea4ee79f492c0495d7f3fce6389034fe26bfb8f3c877f8c03edf914